nixos/modules/hardware/video/nvidia.nix

```diff
@@ -261,7 +261,9 @@ in {
         ];
         boot = {
           blacklistedKernelModules = ["nouveau" "nvidiafb"];
-          kernelModules = [ "nvidia-uvm" ];
+          extraModprobeConfig = ''
```

Comment:
Also maybe add a comment explaining why it's not `kernelModules`. The `modprobe.d.xml` link might be helpful too

Reply:
Comments added